src/SCRIPTS/BF/PAGES/gpspids.lua

```diff
@@ -1,21 +1,59 @@
-local display = assert(loadScript(radio.templateHome.."gpspids.lua"))()
+local template = loadScript(radio.templateHome.."gpspids.lua")
+if template then
+    template = template()
+else
+    template = assert(loadScript(radio.templateHome.."default_template.lua"))()
+end
+local margin = template.margin
+local indent = template.indent
+local lineSpacing = template.lineSpacing
+local tableSpacing = template.tableSpacing
+local sp = template.listSpacing.field
+local yMinLim = radio.yMinLimit
+local x = margin
+local y = yMinLim - lineSpacing
+local inc = { x = function(val) x = x + val return x end, y = function(val) y = y + val return y end }
+local labels = {}
+local fields = {}
+
+if apiVersion >= 1.041 then
+    x = margin
+    y = yMinLim - tableSpacing.header
+    labels[#labels + 1] = { t = "",         x = x, y = inc.y(tableSpacing.header) }
```

Comment:
While I like this change, and I think doing away with the per-screen-size files is a good move, these page definitions now contain a lot of repeated boilerplate.
I think in a next step we should look into ways to simplify or hide the boilerplate, so that the actual page definitions can be reduced to the things that are actually different for every page, without the repetitive elements that are the same.

Reply:
I'll see what I can do